src/Client/OfficeInterop/OfficeInterop.fs

```diff
@@ -74,27 +100,32 @@ module OfficeInteropExtensions =
 
                 columns
 
-            //|> List.iteri (fun colI col ->         
-            //    col
-            //    |> List.iteri (fun rowI stringCell -> 
-            //        let value = 
-            //            if rowI = 0 then
-                    
-            //                match Dictionary.tryGet stringCell stringCount with
-            //                | Some spaces ->
-            //                    stringCount.[stringCell] <- spaces + " "
-            //                    stringCell + " " + spaces
-            //                | None ->
-            //                    stringCount.Add(stringCell,"")
-            //                    stringCell
-            //            else stringCell
-            //        let address = FsAddress(rowI+1,colI+1)
-            //        fsTable.Cell(address, ws.CellCollection).SetValueAs value
-            //    )  
-            //)
-            //ws
-
-    let x = 0
+        static member arcTableFromExcelTableName (tableName:string, context:RequestContext) =
```

Comment:
name can be shortened `ArcTable.arcTableFromExcelTableName` duplicates arctable. Can be shortened to `ArcTable.fromExcelTableName` without loosing information